### ISSUE TITLE
Access Requests: Unnecessary validation when searching for operator [3]

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#1428]
Describe the Bug:

When a user searches for an operator by legal name and nothing is found, they get a message that nothing is found but also a validation error for a required field: https://www.loom.com/share/f1ee191ab071462e8df479b360acfe42

Probability (how likely the bug is to happen, scored from 1-5): 2
(For example, probability of 5 is something like "it happens to all users every time they log in."
Probability of 1 "only happens to certain users when a really specific and unlikely path is followed.")

Effect (how bad the bug is when it does happen, scored from 1-5): 1
(For example, effect of 5 is "the entire app crashes and makes it unusable for all users" or "the bug causes the wrong data to be saved, with critical information (e.g. payment) being affected."
Effect of 1 is "It makes some styling look a little bit weird.")

Steps to reproduce the behaviour:

Screenshots:

Additional information:

- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]
- Additional context

**Comment by Sepehr-Sobhani** [comment]
I don't think this is a bug. It's more of an enhancement if we want to disable the button (Or something like that) until the user selects a valid option from the dropdown.

**Comment by zoeyli-46** [comment]
Hey @BCerki ! We have another ticket to enhance this operator search here: #3128 do you feel like this ticket can be merged with that one? Or close this one?

I would advocate to keep the error message (but reword it, proposed in 3128) instead of disabling the "select operator" button.

**Comment by BCerki** [comment]
Yes, I think we can close this one and use 3128 instead
